subworkflows/nf-core/fasta_newick_epang_gappa/main.nf

```diff
@@ -0,0 +1,156 @@
+include { HMMER_HMMBUILD as HMMER_HMMBUILD          } from '../../../modules/nf-core/hmmer/hmmbuild/main'
```

Comment:
I assume 
```suggestion
include { HMMER_HMMBUILD          } from '../../../modules/nf-core/hmmer/hmmbuild/main'
```
should suffice? But better do not accept this suggestion here because indentation got lost ;)

Reply:
Done.